app/controllers/lists_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
   end
 
   def show
-
+    @list = List.find_by_id(params[:id])
```

Comment:
You should not use params[:id], let's use list_params[:id] and permit the id itself).